controller/jobController.ts

```diff
@@ -1,37 +1,61 @@
 import { Job } from "../model/job";
+import { Login } from "../model/auth";
 import { Responsibility } from "../model/responsibility";
 import { Request, Response, Application } from "express";
+import { SessionData } from "express-session";
 
 const jobService = require("../service/jobService");
+const authService = require("../service/authService");
+const session = require("express-session");
+
+interface CustomSessionData extends SessionData {
+    isLoggedIn: boolean;
+}
+
+declare module "express-session" {
+    interface SessionData {
+        isLoggedIn: boolean;
+    }
+}
 
 module.exports = function(app: Application){
+    app.get("/", async (req: Request, res: Response) => {
+        res.render("index", { isLoggedIn: req.session.isLoggedIn, title: "Home" });
+    });
+
     app.post("/index", async (req: Request, res: Response) => {
         res.redirect("job-roles");
     });
```

Comment:
Why is this needed?

Reply:
@eoinmcbennett This is needed because the Index updates to a different state when the user is logged in. 

This gives the user additional options in the nav bar to choose between Admin or Employee. This code is redirecting from the index to 'job-roles'.

![Screenshot 2024-01-09 at 14 26 02](https://github.com/Cormac-F/infinite-harmony-ui/assets/145450764/fe1bb518-92b1-4b43-a299-40c2041f861b)